README.md

```diff
@@ -90,9 +90,14 @@ Install CSI Driver:
 
 * `--metrics-path`: The HTTP path where prometheus metrics will be exposed. Default is `/metrics`.
 
+* `--retry-interval-start`: Initial retry interval of failed volume snapshot creation or deletion. It doubles with each failure, up to retry-interval-max.
```

Comment:
lets call it "retry-interval-base", _start_ sounds like a starting time. and lets add "<duration>" as resync-period.

Reply:
https://github.com/kubernetes-csi/external-snapshotter/pull/308#issuecomment-634583568